From 0bccfb8271667e4cea4dca01f66da4df786d2fc4 Mon Sep 17 00:00:00 2001
From: Patrick Rudolph <patrick.rudolph@9elements.com>
Date: Wed, 11 Oct 2023 08:13:49 +0200
Subject: [PATCH 08/15] configurations: sbp1: Adjust PVCORE_NIC thresholds

The PVCORE_NIC output voltage is either 0.85V or 0.8V depending
on the SVID lines. Adjust the critical limit to prevent a false
threshold warning.

Change-Id: I576437710bab421414fe1215f46444a55e919521
Signed-off-by: Patrick Rudolph <patrick.rudolph@9elements.com>
---
 configurations/sbp1_baseboard.json | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/configurations/sbp1_baseboard.json b/configurations/sbp1_baseboard.json
index cf8e351..522d01e 100644
--- a/configurations/sbp1_baseboard.json
+++ b/configurations/sbp1_baseboard.json
@@ -594,14 +594,14 @@
                     "Label": "vout1",
                     "Name": "upper critical",
                     "Severity": 1,
-                    "Value": 1.05
+                    "Value": 0.8925
                 },
                 {
                     "Direction": "less than",
                     "Label": "vout1",
                     "Name": "lower critical",
                     "Severity": 1,
-                    "Value": 0.85
+                    "Value": 0.76
                 },
                 {
                     "Direction": "greater than",
@@ -660,14 +660,14 @@
                     "Label": "vout1",
                     "Name": "upper critical",
                     "Severity": 1,
-                    "Value": 1.05
+                    "Value": 0.8925
                 },
                 {
                     "Direction": "less than",
                     "Label": "vout1",
                     "Name": "lower critical",
                     "Severity": 1,
-                    "Value": 0.85
+                    "Value": 0.76
                 },
                 {
                     "Direction": "greater than",
-- 
2.41.0

